sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/CustomShuffleReaderExec.scala

```diff
@@ -76,19 +76,76 @@ case class CustomShuffleReaderExec private(
     val desc = if (isLocalReader) {
       "local"
     } else if (hasCoalescedPartition && hasSkewedPartition) {
-      "coalesced and skewed"
+      s"$coalescedDetail and $skewedDetail"
```

Comment:
It makes sense to add more metrics but it doesn't make sense to include metrics in the plan node string.

Reply:
AFAIK plan node string never contains metrics before this PR.

Reply:
The reason I put this additional info here is that it's easy to see what AQE did in large (100s of operators) plans in the logged version of the plan.  It puts that info in the right context - the 1st image in the PR description shows what it looks like.  
If the additional info was not exposed as SQL metrics but simply computed just to put in the node description - would it still be wrong to put it there?

Reply:
No other plan node does this: metrics are only available through UI or programing APIs (e.g. `QueryExecutionListener`).

I can't find a strong reason to start doing it.

Reply:
remved `stringArgs` changes